pong-game/docker-compose.yml

```diff
@@ -95,8 +95,6 @@ services:
     build:
       context: ./frontend/nginx
       dockerfile: Dockerfile
-    # expose:
-    #   - "80"
```

Comment:
maybe it's due to my macOS not working exactly the same as ubuntu but i can't reach the frontend pages on port 8443 anymore. I land on a 404 Django page